rss/lambda/lambda.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import time
+import json
+import botocore, boto3
+
+# This is stupid, but you can't explicitly import module dependencies.
+sys.path.append('./libs')
+
+import feedparser
+import requests
+
+
+def handler(event, context):
+    db = boto3.resource(
+        "dynamodb",
+        region_name=os.environ["dynamo_region"],
+        config=botocore.client.Config(),
+    ).Table(os.environ["dynamo_table"])
+
+    response = db.query(
+        KeyConditionExpression="id = :id",
+        ExpressionAttributeValues={":id": "lambda.py"},
+    )
+    if len(response["Items"]) == 0:
+        return { "statusCode": 500, "body": json.dumps(f"database not configured"), }
+
+    firstResponse = response["Items"][0]
+    lastUpdateSeen = float(firstResponse["ts"])
+    sharedSecret = firstResponse["ss"]
```

Comment:
Would prefer pep-8 variable names here, even when mirroring keys coming out of some random code (`first_response`, `last_update_seen`, etc.)